linter.py

```diff
@@ -40,7 +40,7 @@
 PLUGINS = {
     'eslint-plugin-html': 'text.html',
     'eslint-plugin-json': 'source.json',
-    'eslint-plugin-svelte3': 'text.html',
+    'eslint-plugin-svelte': 'text.html',
```

Comment:
Please keep the old one as it is just deprecated not gone.

Reply:
Done